docs/src/@primer/gatsby-theme-doctocat/nav.yml

```diff
@@ -70,6 +70,8 @@
       url: /components/markdown
     - title: Octicon
       url: /components/octicon
+    - title: OcticonSymbol
+      url: /components/octiconsymbol
```

Comment:
Is `OcticonSymbolComponent` the best name and is `/components/octiconsymbol` a url that makes sense?